### ISSUE TITLE
feature dim setting with contextWindow

### ISSUE THREAD

**Issue by hspark84** [Microsoft/CNTK#288]
In https://github.com/Microsoft/CNTK/wiki/HTKMLF-Reader, it is described that "dim: an integer that specifies the full feature vector dimension with the desired context window. For example, if you had **72-dimensional features** (24-dimensional filterbank features plus delta and delta-delta coefficients) and the network is designed to process **a context window of 11 frames**, **the specified dimension should be 792**. Note that only symmetric context windows are supported."
And, in code, https://github.com/Microsoft/CNTK/blob/75be1113c596223393baafbecb5b66f64c285ee2/Source/Readers/Kaldi2Reader/HTKMLFReader.cpp  304-line, the feature dim is updated by using the context window information.

I think the wiki page and CNTK book should be modified. Am I right?

**Comment by hspark84** [comment]
What I want to say is that how to set the 'dim' variable in htk/kaldi reader when using contextWindow.

I tired to set the value of dim as full length of the concatenated input vector with explicit contextWindow setting. And I encountered dimension mismatch error between feature and network input.

Then, I read the htkmlfreader.cpp and found that the final feature vector is computed by using the original dimension and contextWindow information.  I think this is not coincident with the wiki description in the above link.

I solved my error by setting the 'dim' in feature as the original size of feature vector.
